packages/plugin-lighthouse/src/lib/utils.unit.test.ts

```diff
@@ -587,3 +615,176 @@ describe('toAuditOutputs', () => {
     expect(outputs[0]?.details).toBeUndefined();
   });
 });
+
+describe('getConfig', () => {
+  it('should return undefined if no path is specified', async () => {
+    await expect(getConfig()).resolves.toBeUndefined();
+  });
+
+  it('should load config from lighthouse preset if preset is specified', async () => {
```

Comment:
🧪 Testing principles: The same function should not be called in one test multiple times.

Suggestion: Splitting it into two tests, one of which could potentially be dynamic.
```
  it.each([
    ['desktop', { formFactor: 'desktop' }],
    ['perf', { onlyCategories: ['performance'] }],
  ] satisfies readonly ['desktop' | 'perf' | 'experimental', object][])(
    'should load config from lighthouse preset if %s preset is specified',
    async (preset, settings) => {
      await expect(getConfig({ preset })).resolves.toEqual(
        expect.objectContaining({
          settings: expect.objectContaining(settings),
        }),
      );
    },
  );
```

Reply:
Done